include/vcpkg/base/optional.h

```diff
@@ -110,14 +110,14 @@ namespace vcpkg
             const T& value() const { return this->m_t; }
             T& value() { return this->m_t; }
 
-        private:
-            void clear()
+            void destroy()
```

Comment:
Can we leave this api as private?

Reply:
Why?

Reply:
(Note that this is on OptionalStorage which itself is effectively a private implementation detail of Optional.

Reply:
Reading comprehension fail on my part then; I was just noting that we were adding a very sensitive API (`destroy()`) as public to a class.